templates/arc/remix.config.js

```diff
@@ -3,7 +3,6 @@ module.exports = {
   future: {
     v2_routeConvention: true,
   },
-  ignoredRouteFiles: ["**/.*"],
```

Comment:
We added these as a generic glob, mainly because we wanted to ignore dotfiles (like `.DS_Store`), but also to give people the chance to ignore other files if needed

Does this mean that wouldn't be possible anymore?

Reply:
> Does this mean that wouldn't be possible anymore?

still possible to ignore things (https://github.com/remix-run/remix/pull/5634/commits/fc2090cc0e923e4b5b5f74bc22ed1635240aef37), but in flat-routes only actual route files are picked up https://github.com/remix-run/remix/pull/5634/files#diff-1047822594960dda7f26a890c6fa53668b6294d6349ee53bcba6590653ce077eR106-R122

all in all, in a flat routes world there should be less need to manually ignore things